common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -575,17 +575,11 @@
             " -Djava.awt.headless=true" +
             " -Dsun.net.inetaddr.ttl=${networkaddress_cache_ttl}" +
             " -Dorg.apache.jasper.compiler.disablejsr199=true" +
-            " -XX:+UseConcMarkSweepGC" +
+            " -XX:+UseG1GC" +
```

Comment:
Do we need perf testing of different garbage collection strategies?

Reply:
Sure we have done with G1GC, will do  with other GC too..